tests/Spout/Reader/XLSX/ReaderTest.php

```diff
@@ -462,6 +462,24 @@ public function testReadShouldCreateOutputEmptyCellPreserved()
 
 
     /**
+     * https://github.com/box/spout/issues/195
+     * @return void
+     */
+    public function testReaderShouldNotTrimCellValues()
+    {
+        $allRows = $this->getAllRowsForFile('sheet_with_untrimmed_strings.xlsx');
```

Comment:
can you rename the file 'sheet_with_untrimmed_inline_strings.xlsx'? So that we'll know it uses inline strings and not shared strings